clang/include/clang/Basic/LangOptions.def

```diff
@@ -543,8 +543,8 @@ BENIGN_LANGOPT(CheckConstexprFunctionBodies, 1, 1,
 LANGOPT(BoundsSafety, 1, 0, "Bounds safety extension for C")
 /* TO_UPSTREAM(BoundsSafety) ON*/
 LANGOPT(BoundsSafetyAttributes, 1, 0, "Experimental bounds safety attributes")
-LANGOPT(BoundsAttributesCXXExperimental, 1, 0, "Experimental bounds attributes for C++")
-LANGOPT(BoundsAttributesObjCExperimental, 1, 0, "Experimental bounds attributes for Objective-C")
+LANGOPT(BoundsSafetyCXXExperimental, 1, 0, "Experimental bounds attributes for C++")
```

Comment:
@hnrklssn @patrykstefanski @rapidsna Should we drop the `Experimental` suffix from the language option? Upstream doesn't seem to do this but I kind of like having the suffix there because it makes it much clearer that this language option is experimental.

Reply:
I have no strong opinion in either direction on this one